packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -325,3 +319,73 @@ function getUpdatedWorkspaceWindowState(workspaceAtTop: OpenWorkspace) {
 export function resetWorkspaceStore() {
   getWorkspaceStore().setState(initialState);
 }
+
+export function closeAllWorkspaces(onClosingWorkspaces: () => void = () => {}) {
```

Comment:
Don't add stuff below `resetWorkspaceStore`; functions that are "just for testing" should be at the bottom of the file. Maybe we should underscore-prefix them, too.

Reply:
Got it, will make the change